### PR TITLE
Following best practices for closing accounts

### PR DESCRIPTION
**Pull request by blockiosaurus**

### CODE REVIEW

**Review by danenbm — APPROVED** [review]

File: core/rust/utils/src/account.rs [hunk 1]
```diff
@@ -111,8 +110,6 @@ pub fn close_account_raw<'a>(
         .unwrap();
     **src_account_info.lamports.borrow_mut() = 0;
```
Comment (danenbm) [inline core/rust/utils/src/account.rs:111]: Nit: if you had time the panicking functions could be replaced:

replace borrow_mut() with try_borrow_mut() and
remove the unwrap()

**Review by danenbm — DISMISSED** [review]
LGTM but left one comment for potential to improve error handling but not required

File: core/rust/utils/src/account.rs (hunk 1 above)
Comment (blockiosaurus) [reply core/rust/utils/src/account.rs:111]: Good call. I couldn't change to a more specific error for numerical overflow without changing the function signature so I picked InvalidRealloc. Maybe in a major version we can add an error argument.

Comment (danenbm) [reply core/rust/utils/src/account.rs]: ah yeah, i think ProgramError ok too for this lib but yeah can change in the future if we like.  Thank you for the fix!

**Review by danenbm — APPROVED** [review]

**Pull request merged**